ocaml/runtime4/interp.c

```diff
@@ -1231,7 +1231,7 @@ value caml_interprete(code_t prog, asize_t prog_size)
 
 #ifndef THREADED_CODE
     default:
-#if _MSC_VER >= 1200
+#if defined(_MSC_VER) && _MSC_VER >= 1200
```

Comment:
We should probably have the same in `ocaml/runtime`.

Reply:
It's just `#ifdef MSC_VER` in 5.2.0 so I don't think we need any runtime5 fix here.  I've harmonized runtime4 with this though.